### ISSUE TITLE
Input with text suffix: VoiceOver reads suffix even though aria-hidden is present

### ISSUE THREAD

**Issue by JoviGriske** [Legal-and-General/canopy#1069]
## Prerequisites

* [x] I have checked the [open Issues](https://github.com/Legal-and-General/canopy/issues) to see if this has already been raised
* [x] I have searched the [closed Issues](https://github.com/Legal-and-General/canopy/issues?q=is%3Aissue+is%3Aclosed) to see if a similar bug has been fixed in the past

## Description

This bug affects:

* [ ] GitHub workflows
* [x] Component
* [ ] Config
* [ ] Performance
* [ ] Tooling / Frameworks
* [ ] Documentation
* [ ] Something else

### Steps to Reproduce

1. In Safari, open up Canopy component: Input with text suffix
2. Start VO
3. Tab into the input field with the text suffix

#### Expected behaviour:

 '%' is not announced due to aria-hidden 

#### Actual behaviour:

 '%' is announced even though aria-hidden is present 

#### Recurrence frequency:

* [x] Occurs every time
* [ ] Occurs intermittently
* [ ] Unsure

## Package version

* [ ] This problem started happening recently
* [ ] This problem started happening a while ago
* [x] Unsure

## Configuration and environment

* [ ] This pertains to **local development**
* [ ] This pertains to **the CI**
* [x] This pertains to **Storybook**

## Additional Information

Working example can be found here: https://designsystem.digital.gov/components/input-prefix-suffix/

**Comment by elenagarrone** [comment]
Hey @JoviGriske 👋 
Thanks for creating the issue 👍 

Just to recap, from the link you shared, the behaviour should be:
![Screenshot 2022-10-26 at 12 27 00](https://user-images.githubusercontent.com/8397116/198014682-be2188e6-737b-4822-ad39-3d9eb8a66b80.png)

So we need to change the code so that the suffixes and prefixes have an `aria-hidden` attribute.

I would think this will be a breaking change as applications will have to update their form labels to add a more descriptive label.

**Comment by st3fun1** [comment]
Hey @elenagarrone,

as you said, we have to add an `aria-hidden` attribute directly on the **<lg-suffix>**/**<lg-preffix>** using **@HostBinding** or some other approach. The underlying issue is that the directive is slotting in the content  in its template and, even if you add aria-hidden to that content, the screen reader doesn't take it into account, and it still reads the content.

**Comment by elenagarrone** [comment]
Added a discussion for possible solution: https://github.com/Legal-and-General/canopy/discussions/1082

**Comment by pauleustice** [comment]
@AndreiRosu To pick up again and catch up on what needs doing for this
